scripts/prepdocs.py

```diff
@@ -301,6 +307,12 @@ def remove_from_index(filename):
         # It can take a few seconds for search results to reflect changes, so wait a bit
         time.sleep(2)
 
+# refresh open ai token every 5 minutes
+def refresh_openai_token():
+    if OPEN_AI_TOKEN_CACHE[CACHE_KEY_TOKEN_TYPE] == 'azure_ad' and OPEN_AI_TOKEN_CACHE[CACHE_KEY_CREATED_TIME] + 300 < time.time():
```

Comment:
I'm bad with time math, but are you sure this should be less than? I keep thinking it should be >

Reply:
yes.
assume token created time 00:00
Upto 00:05, we dont refresh token

We refresh token once created time lapses 5 minutes.